Battle of Mobei and Han-Xiognu war (#121)
TheShadowOfHassen/0-ad-history-encyclopedia-mod: unused/Battle_of_Mobei.md

> The Battle of Mobei took place in 119 BC in the Northern Gobi Desert. It was fought between the Han and Xiognu, who at this point had been in a century-long conflict and fought a decade-long war. The Han army commanded by Wei Qing was much smaller and consisted of comparably inexperienced soldiers because another much more powerful Han army had been sent to invade Xiognu territory as well. And the plan was for only this other army to engage with the main Xiognu forces.

Reviewer: ```suggestion
The Battle of Mobei took place in 119 BC in the Northern Gobi Desert. It was fought between the Han and Xiognu, who at this point had been in a century-long conflict and fought a decade-long war. The Han army commanded by Wei Qing was much smaller and consisted of inexperienced soldiers because another more powerful Han army had been sent to invade Xiognu territory in a different location. And the plan was for only this other army to engage with the main Xiognu forces.
```

> The Battle of Mobei took place in 119 BC in the Northern Gobi Desert. It was fought between the Han and Xiognu, who at this point had been in a century-long conflict and fought a decade-long war. The Han army commanded by Wei Qing was much smaller and consisted of inexperienced soldiers because another more powerful Han army had been sent to invade Xiognu territory in a different location. And the plan was for only this other army to engage with the main Xiognu forces.

Reviewer: ```suggestion
The Battle of Mobei took place in 119 BC in the Northern Gobi Desert. It was fought between the Han and Xiognu, who at this point had been in a century-long conflict and a decade-long war. The Han army commanded by Wei Qing was much smaller and consisted of inexperienced soldiers because another more powerful Han army had been sent to invade Xiognu territory in a different location. And the plan was for only this other army to engage with the main Xiognu forces.
```

> After over 500 miles of exhausting marching, however, Wei Qing's weaker army suddenly stumped into a Xiognu army. It was 80,000 elite cavalry, commanded by the Shanyu, the Xiognu supreme ruler, the entire empire's military pride, well-rested and battle-ready.

Reviewer: ```suggestion
After over 500 miles of exhausting marching, however, Wei Qing's weaker army suddenly met a Xiognu army. It was 80,000 elite cavalry, commanded by the Shanyu, the Xiognu supreme ruler, the entire empire's military pride, well-rested and battle-ready.
```

> After over 500 miles of exhausting marching, however, Wei Qing's weaker army suddenly stumbled into a Xiognu army. It was 80,000 elite cavalry, commanded by the Shanyu, the Xiognu supreme ruler, the entire empire's military pride, well-rested and battle-ready.

Reviewer: ```suggestion
After over 500 miles of exhausting marching, however, Wei Qing's weaker army suddenly stumbled into a much larger Xiognu force. It was 80,000 elite cavalry, commanded by the Shanyu, the Xiognu supreme ruler, each one of them well-rested and battle-ready.Upon sighting the Han, the Xiognu immediately sent cavalry to charge at the disorganized and totally unprepared Han.
```

> Upon sighting the Han, they immediately sent cavalry to charge at the disorganized and totally unprepared Han.

Reviewer: ```suggestion
Upon sighting the Han, the Xiognu immediately sent cavalry to charge at the disorganized and totally unprepared Han.
```

> Upon sighting the Han, the Xiognu immediately sent cavalry to charge at the disorganized and totally unprepared Han.

Reviewer: ```suggestion
```

> The surprised and shocked Wei Qing acted fast, ordering his men to form a ring formation with their chariots, wagons, and carts. These functioned as fortifications to stop the Xiognu cavalry charge. Behind stood the dangerous Han ranged units, archers, and crossbowmen, continuously firing at the enemy.

Reviewer: ```suggestion
Although surprised, Wei Qing acted fast, ordering his men to form a ring formation with their chariots, wagons, and carts. These functioned as fortifications to stop the Xiognu cavalry charge. Behind them stood the dangerous Han ranged units, archers, and crossbowmen, continuously firing at the enemy.
```

> And this tactic worked well: the Xiognu were unable to breach the defensive Han line. A stalemate ensued, and fighting continued until night. In the midst of the night, a sandstorm approached. Wei Qing sent his own cavalry to flank the enemy on both sides. Because of the low visibility due to the night and sandstorm, Han cavalry was able to sneak around the two clashing armies and attack from behind. The Xiognu, unable to coordinate a reaction, panicked. Many of their soldiers were slaughtered, and even though the Shanyu himself managed to get away, the Xiognu army was decisively defeated and, with that, had lost the war.

Reviewer: ```suggestion
And this tactic worked well: the Xiognu were unable to breach the defensive Han line. A stalemate ensued, and fighting continued until night. In the middle of the night, a sandstorm approached. Wei Qing sent his cavalry to flank the enemy. Because of the low visibility due to the night and sandstorm, Han cavalry was able to sneak around the two clashing armies and attack from behind. The Xiognu, unable to coordinate a reaction, panicked. Many of their soldiers were slaughtered, and even though the Shanyu himself managed to get away, the Xiognu army was decisively defeated and, with that, had lost the war.
```
